### DISCUSSION TITLE
v-field-select not rendering anymore

### DISCUSSION THREAD

**Discussion by shadowcpy** [directus/directus#12504] [Archive]
After update to v9.7.1, in my Directus Extension `v-field-select` is not rendering anymore with the following configuration:
```
<v-field-select
    :collection="selected_collection"
    v-model="selected_fields"
    depth="1"
    class="center-export"
/>
```
```
data() {
  return {
    selected_collection: "teilnehmer",
    selected_fields: ["id"]
  };
}
```
```
.center-export {
  display: flex;
  justify-content: center;
  align-items: center;
  margin-left: auto;
  margin-right: auto;
  text-align: center;
}
```

The achieved result looks like this:
![image](https://user-images.githubusercontent.com/65440078/161040026-3272711d-30a4-40b2-90de-2fedf2b4e408.png)
![image](https://user-images.githubusercontent.com/65440078/161040110-0a88646f-4f10-4e4e-84b7-486e0854ac6f.png)

<details>
<summary>More Info</summary>

### To Reproduce

-

### Errors Shown

No errors
![image](https://user-images.githubusercontent.com/65440078/161040194-8a82f208-d50a-4b95-8951-7d9ec9c94b63.png)

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

v14.17.5

### What database are you using?

Postgres 13.5

### What browser are you using?

Brave

### What operating system are you using?

Arch Linux

### How are you deploying Directus?

Remote Linux Server / Baremetal

</details>

**Comment by azrikahar** [comment]
Turns out the component `v-field-select` itself has been removed in #12154 and replaced by `v-field-list` instead. [Link to `v-field-list` component](https://github.com/directus/directus/blob/main/app/src/components/v-field-list/v-field-list.vue)

I'm not sure will it be possible to make the experience 1:1 with what you had without too much changes, but notable modifications might be turning it to: `<v-field-list :collection="selected_collection" @select-field="selectField" />` with the `selectField` function getting the field selected rather than using v-model.

**Comment by shadowcpy** [comment]
Okay, thanks. It is however suboptimal that such big removals are not even listed in the release notes and that the doc page for the component is still up. Does this mean that it is best to do a 100% manual diff review of the code before updating directus?

**Reply by azrikahar** [reply]
> It is however suboptimal that such big removals are not even listed in the release notes and that the doc page for the component is still up.

Totally share the sentiment here. I've removed the doc page in the wiki now to avoid further confusions, and I'll try to bring it up as a discussion internally and find out what is the consensus when such a change is to happen again in the future. Will give an update on it once I have it 👍

**Reply by benhaynes** [reply]
I completely agree, @m0rtis0 — this should have been clearly communicated in the Release Notes. We'll discuss this internally to understand why this didn't happen and will ensure we do better moving forward.

**Reply by shadowcpy** [reply]
@azrikahar @benhaynes Thanks!

**Reply by benhaynes** [reply]
Just CC'ing @erondpowell here, who will be adding some Docs to the beginning of our Extensions section that will clarify all this. Thanks again for pointing this out!
